### ISSUE TITLE
Components should have minimal height, not fixed

### ISSUE THREAD

**Issue by PavelHolec** [kiwicom/orbit-swiftui#107]
Components (such as `Button`) should use `Strut`s, instead of fixed `frame(height:)` to allow growing with bigger dynamic size fonts.
